### DISCUSSION TITLE
Example of launch.json when setup vscode debugger

### DISCUSSION THREAD

**Discussion by zigang93** [vercel/turborepo#1974] [Help]
Anyone can share your launch.json to setup your turborepo debug

**Comment by mehulkar** [comment]
Could you add some more information about what you're trying to do here?

**Reply by corbinday** [reply]
I am wondering the same thing. In my case, my monorepo has a couple nextjs apps and a few local packages. The apps use my local packages, mostly in server-side functions. I would like to debug code in my packages when they are called by the app. 

Currently, when I set breakpoints in my package code and start the debugging process with the [recommended nextjs launch.json](https://nextjs.org/docs/pages/building-your-application/configuring/debugging), the breakpoints are greyed out and say they're unbound.
